tasks/deploy-redis.js

```diff
@@ -4,24 +4,21 @@ const os = require('os');
 const util = require('util');
 const gutil = require('gulp-util');
 
-const {
-  env,
-  getRevision,
-  getConfigFor,
-  getRedisClient,
-} = require('./utils');
+const { env, getRevision, getConfigFor, getRedisClient } = require('./utils');
 
 //
 // Deployment process tasks
 //
 
-async function uploadFile(config, file, rev) {
+async function uploadFiles(config, files, rev) {
   const client = await getRedisClient(config);
 
   const timestamp = new Date();
 
-  // Store index file under indexKey, e.g. 'app:<rev-number>'
-  await client.set(util.format(config.indexKey, rev), file);
+  // Store file under appropriate keys, e.g. 'app:<rev-number>'
+  for (let fileKey in files) {
+    await client.set(util.format(fileKey, rev), files[fileKey]);
+  }
```

Comment:
To efficiently handle a list of async operations you should use `Promise.all`. For reference see https://github.com/productboard/webpack-deploy/pull/45/files#diff-65822c82e442afc2b34df09c3ad934a0R50

Reply:
I see, you want to run it in parallel, right?

Reply:
Yes